src/webui/webapplication.cpp

```diff
@@ -394,6 +394,7 @@ void WebApplication::action_command_download()
     QStringList list = urls.split('\n');
     bool skipChecking = request().posts["skip_checking"] == "true";
     bool addPaused = request().posts["paused"] == "true";
+    bool hasRootFolder = request().posts["root_folder"] == "true";
```

Comment:
I would use reversed option "Strip root folder" everywhere since that's exactly what we're doing. The torrent has root folder initially, but we prevent its creation on disk.

Reply:
But that's how it is in the GUI currently and i'm too lazy to search and figure out how to change that...

Reply:
Correct, I tested the Windows 3.4.0beta2 and the default state is **[x] Create subfolder** (ticked), which I believe this webui setting mirrors that same existing behavior